src/components/PaginatorBar.jsx

```diff
@@ -64,7 +64,7 @@ export const PaginatorBar = (props) => {
               num => (
                 <PaginatorButton
                   key={num}
-                  props={{ className: `pagination-btn ${currentPage === num ? 'active' : ''}`, ...getPageItemProps({ pageValue: num, onPageChange: props.onPageChange }) }}
+                  props={{ className: `pagination-btn ${currentPage === num ? 'active' : ''}`, 'aria-current': currentPage === num ? 'page' : undefined, 'aria-label': `Page ${num}${currentPage === num ? ', current page' : ''}`, ...getPageItemProps({ pageValue: num, onPageChange: props.onPageChange }) }}
```

Comment:
I'm having a hard time parsing what this does - it can really use some simplification or extraction to a utility method that is easier to maintain.

Reply:
I started looking into refactoring it and did some work here, however, it looks like this component, `PaginatorBar`, has zero usages in the codebase. We should probably remove this file altogether.